packages/utilities/src/utils/get-examples-for-schema.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright 2017 - 2023 IBM Corporation.
+ * SPDX-License-Identifier: Apache2.0
+ */
+
+/**
+ * @private
+ */
+const collectFromComposedSchemas = require('./collect-from-composed-schemas');
+
+/**
+ * Returns an array of examples for a simple or composite schema. For each composed schema, if
+ * `schema.examples` is present (and an array), `schema.example` is ignored.
+ * @param {object} schema simple or composite OpenAPI 3.0 schema object
+ * @returns {Array} examples
+ */
+function getExamplesForSchema(schema) {
+  return collectFromComposedSchemas(schema, s => {
+    if (Array.isArray(s.examples)) {
```

Comment:
I'm under the impression that `examples` is only supported on Parameter and Media Type objects, not on schemas generally, and that it's a map. I thought only `example` was supported on schemas - does the spec allow for a list of examples as well?

Reply:
OpenAPI 3.1 [deprecates `example`](https://spec.openapis.org/oas/v3.1.0#fixed-fields-19) in favor of ["native" JSON Schema `examples`](https://json-schema.org/draft/2020-12/json-schema-validation#name-examples).

Reply:
Wow, I'm not sure how I got mixed up there - I thought `example` was the native field from JSON Schema. I probably need to update some things in my new rule based on that knowledge, as well. Thanks for linking that!